src/main/java/frc/robot/subsystems/conditioners/ComposedConditioner.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.subsystems.conditioners;
+
+public class ComposedConditioner implements DriveConditioner {
+
+  private final DriveConditioner[] conditioners;
+
+  public ComposedConditioner(DriveConditioner... conditioners) {
+    this.conditioners = conditioners;
+  }
+
+  @Override
+  public double conditionMove(double moveRequest) {
+    double value = moveRequest;
+    for (int i = 0; i < conditioners.length; i++) {
```

Comment:
This could be written a bit more concisely as:
for (DriveConditioner cond : conditioners)
  value = cond.conditionMove(value);